src/rest/rest-api.php

```diff
@@ -55,6 +55,13 @@ public static function permission_callback() {
 	}
 
 	public static function get_block_content( $params ) {
+		if ( isset( $params[ 'skip_blocks' ] ) && isset( $params[ 'include_blocks' ] ) ) {
+			return new WP_Error( 'vip-block-data-api-invalid-params', 'Cannot use both skip_blocks and include_blocks parameters', [ 'status' => 400 ] );
+		}
+
+		$blocks_to_skip = isset( $params[ 'skip_blocks' ] ) ? explode( ",", $params[ 'skip_blocks' ] ) : [];
```

Comment:
Make sure to also set up validate/sanitize functions for these parameters in the `register_rest_route()` call above under `'args'`.

Reply:
Yip, agreed. Done